.buildkite/nightly-benchmarks/README.md

```diff
@@ -119,6 +133,30 @@ If you do not see the table, please wait till the benchmark finish running.
 The json version of the table (together with the json version of the benchmark) will be also attached to the markdown file.
 The raw benchmarking results (in the format of json files) are in the `Artifacts` tab of the benchmarking.
 
+The `compare-json-results.py` helps to compare benchmark results json files converted using `convert-results-json-to-markdown.py`.
```

Comment:
Why results need to be converted to MarkDown, to do a JSON comparison???
```suggestion
The `compare-json-results.py` helps to compare benchmark results JSON files converted using `convert-results-json-to-markdown.py`.
```

Reply:
@eero-t  you will have both benchmark_results.json and benchmark_results.md files after you run the convert-results-json-to-markdown.py.   json-to-markdown means many json files from different tests into a single md file.